### PR TITLE
Create Dockerfile and Docker Rake tasks

### PR DESCRIPTION
**Pull request by Thor77**
[TURNIERE-108](https://jiratinf16b4.it.dh-karlsruhe.de:8443/browse/TURNIERE-108)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.6.2-alpine
+RUN apk add build-base tzdata sqlite sqlite-dev && gem install tzinfo-data
+WORKDIR /app
+COPY Gemfile /app/Gemfile
+COPY Gemfile.lock /app/Gemfile.lock
+RUN bundle install --deployment --without development test
+COPY . /app
```
Comment (Malaber) [inline Dockerfile]: Do we really want the whole git repository within our app? It would make the container unnecessary big

**Review by Malaber — CHANGES_REQUESTED** [review]
I would specify more files/folders in the .dockerignore to make the image as small as possible

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.git
```
Comment (Malaber) [inline .dockerignore:1]: Add more ignores files here
```suggestion
.git
test/**
spec/**
tmp/**
.gitignore
.hound.yml
.rspec
.rubocop.yml
.travis-yml
docker-compose.yml
README.md
```

**Review by Malaber — CHANGES_REQUESTED** [review]
CHANGE!

File: .dockerignore (hunk 2 above)
Comment (Thor77) [reply .dockerignore:1]: `749MB` vs `717MB` :tada:

**Review by Malaber — APPROVED** [review]

**Pull request merged**